README.md

```diff
@@ -29,3 +29,20 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons of pancakes? What is it in terms of the number of flips?
 Add your answer to this markdown file.
 
+So my implementation has three nested loops at its center. The way i see this sort working is that it tries to seperate the high numbers from the low numbers and with enough attempts they graduly seperate into the right order. I bet it looks adjacent to bubble sort. Im going to assume though this logic that this algorithm can atleast find 1 correctly placed element per loop, meaning that the outer loop runs in the worst case n times. The next loop iterates though every element in the array and follows some not so easy logic that i made up. So obiously it also runs n ammount of times. Then finally my flip implementation is just bad and iterates though every element before the spatula 2 times, first to put it in an array and second to get it back out, in the worste case this will happend the length of the array, so 2n times for the flip function.
+
+In the end my pancake sort implementation is T(n) = 2n * n * n $\in$ $\theta(n^3)$
+
+Comparisons:
+outer loop runs atmost n times and the inner loop runs n times aswell, at the end of these are always the comparisons making the comparisons:
+
+T(n) = n * n $\in \theta(n^2)$
+
+Flips:
+the flips happend whenever a loop condition is met and in the worst case there would be a flip every comparison making it the same as the comparions:
```

Comment:
Could you avoid making this many flips? In particular you could figure out where a particular pancake goes and then only make a few flips to get it there.